mitodl/ocw-studio
# Changed WebsiteStarter.config to store JSON instead of YAML

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Changes `WebsiteStarter.config` property so that it stores JSON instead of raw YML

#### How should this be manually tested?
Migrate, check the `config` values for some starters in Django admin, and try updating them

#### Any background context you want to provide?
- Our own APIs are going to be passing this config around as JSON, so it doesn't make much sense to store it as YML. That was an oversight on my part during the first implementation
- I added an admin class for pretty-printing JSONField-type fields

#### Screenshots (if appropriate)
![ss 2021-02-05 at 14 12 36 ](https://user-images.githubusercontent.com/14932219/107241715-3f050980-69f9-11eb-8730-4171728ea94e.png)
